source/_docs/guides/github-pull-requests.md

```diff
@@ -0,0 +1,340 @@
+
+    ```bash
+    terminus drush $SITE.$ENV -- pm-enable pathauto --yes
+    ```
+
+3.  In Drupal 8, the set of enabled modules is also part of the exportable configuration set. That means we can track enabled modules in Composer with the  **Update** tab in the **Configuration Synchronization** section of the Drupal admin interface, as we did in step 2 of the [Configure Your Site Through Drupal's Admin Interface](#configure-site-via-the-admin-interface) section. Alternately, this same operation may be done from the command line using Terminus and Drush:
+
+    ```bash
+    terminus drush $SITE.$ENV -- config-export --yes
+    ```
+
+4.  You can also commit your changes from the command line with Terminus:
+
+    ```bash
+    terminus env:commit $SITE.$ENV --message="Install and enable pathauto"
+    ```
+
+    The information needed to install and enable pathauto and its dependencies is now committed to your GitHub repository. The modules sources themselves, however, will not be part of this commit.
+
+## Create a Custom Theme
+
+A custom theme is the most convenient way to add css styles or alter the markup produced by modules. Most Drupal sites will have a custom theme to differentiate the appearance of the site. For comprehensive documentation on how to create themes for Drupal 8, see [Theming Drupal 8](https://www.drupal.org/docs/8/theming) on drupal.org.
+
+A simple theme may be created quite easily using Drupal Console. The example below creates a subtheme of [Bartik](https://www.drupal.org/project/bartik) to allow for simple css changes for demonstration purposes. Note that usually, themes will subclass either `classy` or `stable`, which provide a blank slate to start from.
+
+1.  Run the `generate:theme` command as shown below to start the process of creating a subtheme.
+
+    ```
+    terminus drupal $SITE.$ENV -- generate:theme
+    ```
+    Drupal Console will ask a series of questions about your theme. For many of them you can use the default value by just hitting **Enter**. Answer these quesions as follows:
+
+    - Theme: Amazing Theme
+    - Base theme: bartik
+    - Enter 'no' to generate theme regions and theme breakpoints
+
+    Type `yes` when asked to confirm generation of the theme. Once you do this, the files for your new theme will be written to the directory `code/web/themes/custom/amazing_theme`. If you gave your theme a different name, replace `amazing_theme` with the appropriate name for your theme.
+
+2.  Use an SFTP client to access the generated theme files in [Pantheon's on-server development mode](https://pantheon.io/docs/sftp/). Find the file `code/web/core/theme/bartik/bartik.info.yml`, and open it in an editor. Copy the regions section starting on line 28, and paste it into the file `code/web/themes/custom/amazing_theme/amazing_theme.info.yml`. Optionally, you might also want to copy the file `code/web/core/theme/bartik/logo.svg` into your theme folder.
```

Comment:
The `regions` section of Bartik will not necessarily start on line 28 if changes are made in the future. This file probably will not change often, so it's up to you whether you think this is worth changing or not.